recipes-kernel/kernel-tests/kernel-performance-tests-files/run-cyclictest

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
```

Comment:
Could you set the shebangs to `/bin/bash` only, since you're using non-POSIX syntax in most of the scripts.

Reply:
OK